web/src/js/flow/utils.js

```diff
@@ -49,7 +49,14 @@ export var MessageUtils = {
         } else if (message === flow.response) {
             message = "response";
         }
-        return `/flows/${flow.id}/${message}/content` +  (view ? `/${view}` : '');
+        if (global.MITMWEB_STATIC) {
```

Comment:
Our backend could not redirect the url like `/flows.json` to `/flows`, it will return a 404. Is there any setting that can help us with this?